client/plots/controls.config.js

```diff
@@ -418,7 +418,8 @@ function setRadioInput(opts) {
 				}
 		  ]
 
-	const styles = opts.styles
+	const styles = opts.styles || {}
+	styles['display'] = inputs.length > 2 ? 'block' : 'inline-block'
```

Comment:
This style change could affect a lot of other plot control menus, should verify the radio input displays still looks the same for other plots. Or, just specify `'opts.styles['display'] = 'block' | 'inline-block'` in your plot controls menu, so as not to affect other plots.